backend/api/Areas/Tools/Helpers/ImportProjectsHelper.cs

```diff
@@ -108,45 +140,117 @@ private Entity.Project Merge(Entity.Project project, Model.ImportProjectModel mo
+                project.StatusId = status.Id;
+
+                // Extract properties from PID note.
+                var pids = model.Notes?.FirstOrDefault(n => n.Key == "PID").Value;
+                if (!String.IsNullOrWhiteSpace(pids))
+                {
+                    var matches = Regex.Matches(pids, "[0-9]{3}-[0-9]{3}-[0-9]{3}");
+
+                    // Need to load any properties currently linked to this project.
+                    if (project.Id > 0)
+                    {
+                        var existingProject = _service.Project.Get(project.ProjectNumber);
+
+                        matches.ForEach(m =>
+                        {
+                            // If the parcel has not already been added, add it to the project.
+                            if (!existingProject.Properties.Any(p => p.PropertyType == Entity.PropertyTypes.Land && p.Parcel.ParcelIdentity == m.Value))
+                                matches.ForEach(m => AddProperty(project, m.Value));
+                        });
+
+                    }
+                    else
+                    {
+                        matches.ForEach(m => AddProperty(project, m.Value));
+                    }
+                }
+
+                project.TierLevel = GetTier(model.Market, project.Properties.Count()); // TODO: Need to import or link project properties.
                 project.TierLevelId = project.TierLevel.Id;
                 project.Risk = GetRisk(model.Risk);
                 project.RiskId = project.Risk.Id;
 
-                project.OcgFinancialStatement = model.OcgFinancialStatement;
+                project.Assessed = model.Assessed ?? project.Properties.Where(p => p.PropertyType == Entity.PropertyTypes.Land).Sum(p => p.Parcel.Evaluations.Where(e => e.Key == Entity.EvaluationKeys.Assessed).OrderByDescending(e => e.Date).FirstOrDefault()?.Value ?? 0);
                 project.NetBook = model.NetBook;
-                project.Estimated = model.Estimated;
-                project.ProgramCost = model.ProgramCost;
-                project.SalesCost = model.SalesCost;
-                project.InterestComponent = model.InterestComponent;
-                project.NetProceeds = model.NetProceeds;
-                project.GainLoss = model.GainLoss;
-                project.SaleWithLeaseInPlace = model.SaleWithLeaseInPlace;
-
-                if (model.PriorNetProceeds.HasValue)
-                {
-                    project.Snapshots.Add(new Entity.ProjectSnapshot(project) { NetProceeds = model.PriorNetProceeds.Value });
-                }
+                project.Market = model.Market;
+                project.Appraised = model.Appraised;
 
-                project.MarketedOn = model.MarketedOn;
+                project.SubmittedOn = new DateTime(model.ReportedFiscalYear - 1, 1, 1); // Don't have a source for this information.
+                project.ApprovedOn = project.SubmittedOn; // Don't have a source for this information.
```

Comment:
double checking that this is correct.

Reply:
`SubmittedOn` is when the project was submitted for assessment.  `ApprovedOn` is when the project was approved by SRES to be added to SPP.  SIS data doesn't have either of these dates in their data regrettably.  So I've simply assumed when it was reported last will be the date.  I'm wondering if I should leave them null.